securedrop/journalist.py

```diff
@@ -581,16 +581,17 @@ def bulk():
 
     doc_names_selected = request.form.getlist('doc_names_selected')
     selected_docs = [doc for doc in g.source.collection
-                     if doc.filename in doc_names_selected]
-
+                     if (doc.filename in doc_names_selected) or action == 'download_all']
```

Comment:
Yes, this line should be less than 80 characters long.

Also, you don't need the parentheses around `(doc.filename in doc_names_selected)`. According to [Python operator precedence](https://docs.python.org/2/reference/expressions.html#operator-precedence), `in` binds more tightly than boolean operators like `or`.